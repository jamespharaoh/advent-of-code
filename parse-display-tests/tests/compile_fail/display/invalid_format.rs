use parse_display::Display;
#[derive(Display)]
#[display("{x")]
struct TestStruct;

fn main() {}
