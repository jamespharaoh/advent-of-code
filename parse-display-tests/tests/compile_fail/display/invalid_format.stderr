error: invalid display format.
 --> $DIR/invalid_format.rs:3:11
  |
3 | #[display("{x")]
  |           ^^^^
